postgres-driver/postgres_driver.go

```diff
@@ -203,12 +333,40 @@ func (d *PostgresDriver) WriteBlock(block *indexer.Block) error {
 	return nil
 }
 
-// ReadBlocks returns all blocks on the database
-// TODO: add pagination
-func (d *PostgresDriver) ReadBlocks() ([]*indexer.Block, error) {
+// ReadBlocksOptions optional parameters for ReadBlocks
+type ReadBlocksOptions struct {
```

Comment:
why you have the same struct values with different names, I think you could use a single one called `GetPaginatedOptions` and use it whenever the only available body values are `perPage` and `page`

Reply:
It's so probable we'll need more optional values for each query, that's why I separated it

Reply:
all right